packages/genui/a2ui-playground/src/pages/AIChatPage.css

```
@@ -92,14 +95,202 @@
+  width: 12px;
+  height: 12px;
+  border: 1.5px solid
+    color-mix(in srgb, var(--geist-secondary) 35%, transparent);
+  border-top-color: var(--geist-secondary);
+  border-radius: 50%;
+  display: inline-block;
+  animation: chatMessageActionSpin 0.8s linear infinite;
+  flex-shrink: 0;
+}
+
+@keyframes chatMessageActionSpin {
```

Stylelint (17.11.1)

[error] Expected keyframe name "chatMessageActionSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)